src/StatusQ/Components/StatusChatListItem.qml

```diff
@@ -28,8 +28,9 @@ Rectangle {
         width: 24
         height: 24
         color: Theme.palette.miscColor5
-        letterSize: emoji ? 14 : 15
+        letterSize: emoji ? 10 : 15
         emoji: ""
+        emojiSize: "16x16"
```

Comment:
Hey I notice in the Emoji.qml, we have predefined emoji sizes 
    readonly property var size: {
        "big": "72x72",
        "middle": "32x32",
        "small": "18x18"
    }
maybe we can added 16*16 here as well?  also by default it was already 18*18 what going wrong visually in the chat items?

Reply:
I can add it to the sizes. The problem was that `18x18;  was too big in the channel list.

Reply:
got it :)

Reply:
Updated